### ISSUE TITLE
CppCheck reports the column too.

### ISSUE THREAD

**Issue by ferkulat** [tomasbjerre/violations-lib#136]
At least in version 2.6 I can see it reports the column too.
A project that makes use of your library plans to make use of the information about the column.

If violations-lib provides this information, jenkins `warnings-ng-pulgin` could provide this in cppcheck reports too.

https://github.com/jenkinsci/warnings-ng-plugin/pull/1129

~~~~
<?xml version="1.0" encoding="UTF-8"?>
<results version="2">
    <cppcheck version="2.6"/>
    <errors>
        <error>
            <location file="C:\Service.cpp" line="19" column="0"/>
            <symbol>showProfilePage</symbol>
        </error>
   </errors>
</results>
~~~~

 Thank you very much

**Comment by tomasbjerre** [comment]
Should be fixed now in `1.146.0`. Open issue again if not working.

**Comment by ferkulat** [comment]
Wow that was quick, I was trying to fix that too :)
Thank you very much
